cves/kernel/CVE-2014-1690.yml

```diff
@@ -456,7 +484,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The biggest thing pointed out about this vulnerability was that its creation
```

Comment:
What made you deem the feature as unneeded?

Reply:
Removed to agree with previous comment.